cli/smartcontract/manifest.go

```diff
@@ -0,0 +1,122 @@
+package smartcontract
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/nspcc-dev/neo-go/cli/flags"
+	"github.com/nspcc-dev/neo-go/pkg/core/state"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/nef"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+	"github.com/urfave/cli"
+)
+
+func manifestAddGroup(ctx *cli.Context) error {
+	walletPath := ctx.String("wallet")
+	if len(walletPath) == 0 {
+		return cli.NewExitError(errNoWallet, 1)
+	}
+
+	w, err := wallet.NewWalletFromFile(walletPath)
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+	defer w.Close()
+
+	addr, err := flags.ParseAddress(ctx.String("account"))
```

Comment:
Let's use an address flag instead of a string?

Reply:
I find it more verbose in case we require non-empty flags.